components/locale/kmr_KU.tsx

```diff
@@ -1,11 +1,11 @@
-import Pagination from 'rc-pagination/lib/locale/ku_IQ';
-import DatePicker from '../date-picker/locale/ku_IQ';
-import TimePicker from '../time-picker/locale/ku_IQ';
-import Calendar from '../calendar/locale/ku_IQ';
+import Pagination from 'rc-pagination/lib/locale/kmr_KU';
+import DatePicker from '../date-picker/locale/kmr_KU';
+import TimePicker from '../time-picker/locale/kmr_KU';
+import Calendar from '../calendar/locale/kmr_KU';
 import { Locale } from '../locale-provider';
 
 const localeValues: Locale = {
-  locale: 'ku-iq',
+  locale: 'kmr-ku',
```

Comment:
It would be a breaking change. We should keep compatible to original usage.

Reply:
yes this PR could remain open untill 
**rc-picker** and **rc-paginate** gets updated i have created PR on those two packages also, after they update we can change dependency for those packages to latest change

Reply:
I understand what `afc163` means, keep `ku_IQ` and add new `kmr_KU`, is that right?

Reply:
@yoyo837 this is an alternative you could keep ku_IQ and make an alias of it named kmr_KU this is a solution also but it should be deprecated in the future